.github/workflows/pythonpackage.yml

```diff
@@ -81,7 +81,7 @@ jobs:
     - uses: actions/checkout@v4
     - name: Set up Python
       uses: actions/setup-python@v5
-    - run: pipx run ruff check mriqc/
+    - run: pipx run ruff format --diff
```

Comment:
This changed `ruff check` into `ruff format`. On purpose?

Reply:
I had missed it. Does format include check?

Reply:
No , `ruff check` and `ruff format` are different. I can add `ruff check` back  in #1348.

Reply:
Also, do we want to apply ruff to `mriqc/` only, or to the whole repository? I think applying ruff to the whole repository was a good move.

Reply:
Thanks for this, yes, please include these suggestions within #1348.